recipes/notcurses/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class NotcursesConan(ConanFile):
+    name = "notcurses"
+    description = "a blingful TUI/character graphics library"
+    topics = ("graphics", "curses", "tui", "console", "ncurses")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://nick-black.com/dankwiki/index.php/Notcurses"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False,
+                       "readline:with_library":"curses"
```

Comment:
Here in ConanCenter we are only building the recipes with their default options values.... so this packageID for `readline` is not going to be available.

Is it strictly required? Does it make sense to change the default value on the requirement?

Reply:
It should also be in validate i'd imagine